src/worker/worker/bots/intelowl_bot.py

```
@@ -0,0 +1,365 @@
+    ) -> tuple[ObservablePayload, str]:
+        observable_type = observable["type"]
+        submitted_at = self._now()
+        try:
+            response = client.send_observable_analysis_request(
+                observable["value"],
+                tlp=tlp,
+                analyzers_requested=self.analyzer_map[observable_type],
+                observable_classification=self.intelowl_classification[observable_type],
+                tags_labels=["taranis-ai"],
+            )
+        except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)